### DISCUSSION TITLE
What's the pre-requisites in terms of permissions needed to get the cost & billing data.

### DISCUSSION THREAD

**Discussion by dillib** [microsoft/finops-toolkit#405] [Q&A] [answered]
Hi, I was able to install the hub and we have couple of subscriptions to analyze and was not sure if we have sufficient permissions to get the billing data, I would appreciate if any one can clarify on the pre-requisites for getting the billing data, and what is the minimal steps and permissions needed in order to configure the tool kit and the permissions needed.

**Comment by flanakin** [comment, marked as answer]
If you're creating an export on a subscription or resource group, you need Cost Management Contributor.

If you're creating an export on an EA billing scope, you must be an **Enterprise Reader**, **Department Reader**, or as an **Account Owner** (aka enrollment account).

If you're creating an export on an MCA billing scope, you must be a **Contributor** on the billing account, billing profile, customer (CSP partners only), or invoice section.

This is covered in the [Cost Management docs](https://learn.microsoft.com/azure/cost-management-billing/costs/tutorial-export-acm-data), but it'd be good to add an abbreviated version to our docs. I'll get that done.
 
Sorry for the delay!
